app/config/routes.php

```diff
@@ -286,6 +286,14 @@
 );
 
 $router->add(
+    '/add-new-category',
```

Comment:
Could we use `/category/add` to provide ability use transparent routing in future?